src/index.ts

```diff
@@ -489,6 +455,15 @@ export function ConfirmMultisigAddress({
   name,
   policyHmac,
   walletConfig,
+}: {
+  keystore: KEYSTORE_TYPES;
+  network: Network;
+  bip32Path: string;
+  multisig: any;
+  publicKey?: string;
+  name?: string;
+  policyHmac?: string;
+  walletConfig?: any;
```

Comment:
```suggestion
  walletConfig?: MultisigWalletConfig;
```

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff